packages/components/text-link/src/TextLink.tsx

```diff
@@ -109,7 +109,18 @@ const TextLink: PolymorphicComponentWithRef<
   }
 
   return (
-    <a {...commonProps} href={href}>
+    <a
+      {...commonProps}
+      onClick={
```

Comment:
so disabled anchor cannot be clicked

Reply:
Didn't we render disabled TextLink as `span` before?

Reply:
no, we didn't

Reply:
Okay, well that would be my suggestion here to get these behaviours automatically.

Reply:
I guess we'll have some typings problems in this case and we'd need to filter out some properties that are related only to `a` tag. I'd rather keep it this way.

Reply:
Oh! I already approved the PR, but this ☝️  seems like a good idea

Reply:
one more argument against it: we would still need to remove `onClick` handler on `span` element.

Reply:
I agree it's pretty bothersome for types 🙁